opm/autodiff/FlowMain.hpp

```diff
@@ -428,6 +428,13 @@ namespace Opm
                 IOConfigPtr ioConfig = eclipse_state_->getIOConfig();
                 ioConfig->overrideRestartWriteInterval(static_cast<size_t>(output_interval));
             }
+
+            // Possible to force initialization only behavior (NOSIM).
+            if (param_.has("nosim")) {
```

Comment:
I would usually not call ParameterGroup::has() but call ParameterGroup::getDefault() instead. Since that is not used here I assume that is because in the absence of any command-line setting, the behavior should be inferred from the deck (as for the output_interval above)?

The only problem with that is that the option becomes hidden (i.e. there will be no OpmLog::note() saying that it looked for "nosim" etc.) and therefore hard to discover. A small improvement would be to use the getDefault() method, getting the default from the IOConfig. I won't *require* that change in this PR though, feel free to merge when upstream goes in.